protected/config/common.php

```diff
@@ -4,5 +4,20 @@
  * @see http://www.yiiframework.com/doc-2.0/guide-concept-configurations.html
  * @see https://docs.humhub.org/docs/admin/advanced-configuration
  */
+
```

Comment:
This file is for HumHub administrators and should remain empty by default.
Also, changes here quickly lead to conflicts in Git :-)

I'm sure we can handle the ENV Module Paths somewhere else.